src/app/payment/paymentMethod/OpyPaymentMethod.tsx

```diff
@@ -0,0 +1,49 @@
+import { PaymentInitializeOptions } from '@bigcommerce/checkout-sdk';
+import React, { useCallback, FunctionComponent } from 'react';
+import { Omit } from 'utility-types';
+
+import { withLanguage, TranslatedString, WithLanguageProps } from '../../locale';
+import { LoadingOverlay } from '../../ui/loading';
+
+import getPaymentMethodName from './getPaymentMethodName';
+import HostedWidgetPaymentMethod , { HostedWidgetPaymentMethodProps } from './HostedWidgetPaymentMethod';
+
+export type OpyPaymentMethodProps = Omit<HostedWidgetPaymentMethodProps, 'containerId'>;
+
+const OpyPaymentMethod: FunctionComponent<OpyPaymentMethodProps & WithLanguageProps> = ({
+    initializePayment,
+    method,
+    language,
+    isInitializing = false,
+    ...rest
+}) => {
+    const initializeOpyPayment = useCallback((options: PaymentInitializeOptions) => initializePayment({
+        ...options,
+        opy: {
+            containerId: 'learnMoreButton',
```

Comment:
Can we move the name of this containerId, to a variable. Since it can break if we change it here but not on line 33

Reply:
good idea!